resources/assets/pages/ShowAction.js

```diff
@@ -0,0 +1,49 @@
+import gql from 'graphql-tag';
+import React, { useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { useQuery } from '@apollo/react-hooks';
+
+import Action from '../components/Action';
+import Shell from '../components/utilities/Shell';
+import MetaInformation from '../components/utilities/MetaInformation';
+
+const SHOW_ACTION_QUERY = gql`
+  query ShowActionQuery($id: Int!) {
+    action(id: $id) {
+      id
+      name
+      campaignId
+    }
+  }
+`;
+
+const ShowAction = () => {
+  const { id } = useParams();
+
+  const { loading, error, data } = useQuery(SHOW_ACTION_QUERY, {
+    variables: { id: Number(id) },
+  });
+
+  if (error) {
+    return <Shell error={error} />;
+  }
+
+  if (loading) {
+    return <Shell title="Action" loading />;
+  }
+
+  const { campaignId, name } = data.action;
+
+  return (
+    <Shell
+      title={name}
+      subtitle={
+        <a href={`/campaign-ids/${campaignId}`}>{`Campaign ${campaignId}`}</a>
```

Comment:
This is really subjective, but it looks a little odd to have a link here to me (the blue on purple).

What if we added this link below the action, like [on the post permalink](https://github.com/DoSomething/rogue/blob/e795e177698df11811d66b8f8673c15a20e99f40/resources/assets/pages/ShowPost.js#L61-L70)?